### PR TITLE
fix(editor): limiting columns in table view to prevent unresponsive UI when opening NDV

### PR DESCRIPTION
**Pull request by cstuncsik**

### CODE REVIEW

File: packages/editor-ui/src/components/mixins/globalLinkActions.ts [hunk 1]
```diff
@@ -39,7 +39,6 @@
 
 			const actionAttribute = clickedElement.getAttribute('data-action');
 			if(actionAttribute && typeof this.availableActions[actionAttribute] === 'function') {
-				e.preventDefault();
```
Comment (mutdmour) [inline packages/editor-ui/src/components/mixins/globalLinkActions.ts:42]: did you test this change with every action it's used?

File: packages/editor-ui/src/plugins/i18n/locales/en.json [hunk 2]
```diff
@@ -361,6 +361,8 @@
 	"dataMapping.jsonHint": "<img src='/static/json-mapping-gif.gif'/> Drag a JSON key onto <b>{name}</b> to map data",
 	"dataMapping.mapKeyToField": "Map '{name}' to a field",
 	"dataMapping.mapAllKeysToField": "Map every '{name}' to a field",
+	"dataMapping.tableView.tableColumnsExceeded": "Some columns are hidden.",
+	"dataMapping.tableView.tableColumnsExceededTooltip": "Your data has more than {columnLimit} columns so some are hidden. Switch to <a data-action='switchToJsonView{paneType}'>JSON view</a> to see all data.",
```
Comment (mutdmour) [inline packages/editor-ui/src/plugins/i18n/locales/en.json:365]: would not using `<i18n> ` cleaner approach than a global action?

File: packages/editor-ui/src/components/mixins/globalLinkActions.ts (hunk 1 above)
Comment (cstuncsik) [reply packages/editor-ui/src/components/mixins/globalLinkActions.ts]: with using `<i18n>` tag for translation this is not needed anymore, rolled back all changes in this file

File: packages/editor-ui/src/plugins/i18n/locales/en.json (hunk 2 above)
Comment (cstuncsik) [reply packages/editor-ui/src/plugins/i18n/locales/en.json]: thanks for the tip, applied

File: packages/editor-ui/src/components/RunDataTable.vue [hunk 3]
```diff
@@ -189,6 +201,8 @@ export default mixins(externalHooks).extend({
 			hoveringPath: null as null | string,
 			mappingHintVisible: false,
 			activeRow: null as number | null,
+			columnLimit: 40,
```
Comment (mutdmour) [inline packages/editor-ui/src/components/RunDataTable.vue:204]: nitpick - magic number should be set as variable on top `MAX_COLUMNS_LIMIT`

**Review by mutdmour — DISMISSED** [review]
Looks good. minor comment, address if you want

File: packages/editor-ui/src/components/RunDataTable.vue (hunk 3 above)
Comment (cstuncsik) [reply packages/editor-ui/src/components/RunDataTable.vue]: so true, please approve again

**Review by mutdmour — APPROVED** [review]

**Comment by janober** [comment]
Got released with `n8n@0.201.0`

**Pull request merged**